app/AvailablePlugin/DuplicateAccountEnroller/Config/Schema/schema.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" ?>
+<!--
+  COmanage Registry Duplicat Account Enroller Plugin Database Schema
+
+  Portions licensed to the University Corporation for Advanced Internet
+  Development, Inc. ("UCAID") under one or more contributor license agreements.
+  See the NOTICE file distributed with this work for additional information
+  regarding copyright ownership.
+
+  UCAID licenses this file to you under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with the
+  License. You may obtain a copy of the License at:
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  @link          http://www.internet2.edu/comanage COmanage Project
+  @package       registry
+  @since         COmanage Registry v4.3.0
+  @license       Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+-->
+<!--
+  While this schema mostly works with database prefixes, foreign key constraints
+  must be specified in raw SQL, which needs the prefixed table name.
+-->
+
+<schema version="0.3">
+  <table name="duplicate_account_enrollers">
+    <field name="id" type="I">
+      <key />
+      <autoincrement />
+    </field>
+    <field name="co_enrollment_flow_wedge_id" type="I">
+      <constraint>REFERENCES cm_co_enrollment_flow_wedges(id)</constraint>
+    </field>
+    <field name="redirect_url" type="C" />
+    <field name="type" type="C" size="32" />
```

Comment:
`type` should have a more descriptive name, eg `identifier_type`.

Reply:
Done